src/science/landuse/landuse3.F90

```diff
@@ -3,7 +3,26 @@ MODULE landuse_variable
   IMPLICIT NONE
 
   SAVE
-
+!*# Overview of landuse3.F90
+!
+! These are three-type landuse-type variables:
+!
+! - `luc%var_x(mland,mvmax)`: variable indexed by "mland" and "mvmax" before landuse change
+! - `luc%var_y(mland,mvmax)`: variable indexed by "mland" and "mvmax" after land use change
+! - `lucmp%var( mp        )`: variable indexed by the patch number from 1 to mp
+!
+The land use change for each variable is applied in the form of a transformation matrix `T` such that:
+! \[$$\textbf{luc%var_y} = \textbf{luc%var_x} \cdot \textbf{T}\]
+!
+! This module will: 
+!
+! 1. map a variable `var` from CABLE into one derived-type variable `luc%var_x`
+! 2. apply land use change (PFT transition, wood harvest and land management)
+! 3. write the value of `var_x` into `var_y`
+! 4. update the value of `mp` and write `luc%var_y` to `lucmp%var`
+!
+! All variables in the restart file and the state variables will be calculated.
+!
```

Comment:
```suggestion
!*# Overview of landuse3.F90
!
! This MODULE applies land use changes for CABLE including:
!
! - PFT transition
! - wood harvest
! - land management
!
!## Method
!
! There are three-type landuse-type variables:
!
! - `luc%var_x(mland,mvmax)`: variable indexed by "mland" and "mvmax" before landuse change
! - `luc%var_y(mland,mvmax)`: variable indexed by "mland" and "mvmax" after land use change
! - `lucmp%var( mp        )`: variable indexed by the patch number from 1 to mp
!
! The land use change for each variable is applied in the form of a transformation matrix `T` such that:
! \[$$\textbf{luc%var_y} = \textbf{luc%var_x} \cdot \textbf{T}\]
!
! This module will: 
!
! 1. map a variable `var` from CABLE into one derived-type variable `luc%var_x`
! 2. apply land use change (PFT transition, wood harvest and land management)
! 3. write the value of `var_x` into `var_y`
! 4. update the value of `mp` and write `luc%var_y` to `lucmp%var`
!
! All variables in the restart file and the state variables will be calculated.
!
```
Just an idea to structure the documentation a bit more. I like to start with a short purpose statement.